lib/experimental-default-theme.json

```diff
@@ -127,6 +127,9 @@
 		"features": {
 			"typography": {
 				"dropCap": false
+			},
+			"colors": {
+				"allowCustom": true
```

Comment:
I'm assuming this file is where we define if the feature is opt-in or opt-out. (Default value provided by Core).

Reply:
Also by reading this, I wonder if it makes more sense to avoid the separation between the top-level "presets" and "features" and just combine them because these are used in conjunction in general.

Components will do things like:

```
const colorsPreset = useEditorFeature( 'colors.preset' );
const allowCustomColors = useEditorFeature( 'colors.allowCustom' );

// Render palette with these options.
```

Reply:
Alternatively we can provide a separate hook `useEditorPreset( 'colors' )` but it's a bit weird.

Reply:
It's a bit weird to me to see the prefix "allow" given cases like "dropCap" when it is implied. I think just `custom` would be fine.

Reply:
Removed the prefix.

Reply:
Updated https://github.com/WordPress/gutenberg/issues/20588 by adding a task to look into either creating a `usePreset('colors')` hook or integrate into the existing `useEditorFeature('color.preset')`.